api/custom_auth/urls.py

```diff
@@ -25,12 +26,13 @@
         CustomAuthTokenLoginWithMFACode.as_view(),
         name="mfa-authtoken-login-code",
     ),
+    path("logout/", TokenDestroyView.as_view(), name="authtoken-logout"),
     path("", include(ffadmin_user_router.urls)),
     path("token/", delete_token, name="delete-token"),
     # NOTE: endpoints provided by `djoser.urls`
     # are deprecated and will be removed in the next Major release
     path("", include("djoser.urls")),
-    path("", include("trench.urls")),  # MFA
-    path("", include("trench.urls.djoser")),  # override necessary urls for MFA auth
+    path("", include("custom_auth.mfa.trench.urls")),  # MFA
+    # path("", include("trench.urls")),  # MFA
```

Comment:
Is there a reason that this commented out? If not, let's remove it. If there is a reason for this to still be here, let's add a comment explaining why it's commented out.

Reply:
Good catch! I forgot to remove this after our discussion around removing endpoints that are not used by UI